src/targets/gpu/include/migraphx/gpu/rocblas.hpp

```diff
@@ -3,7 +3,7 @@
 
 #include <migraphx/manage_ptr.hpp>
 #include <migraphx/config.hpp>
-#include <rocblas.h>
+#include <rocblas/rocblas.h>
```

Comment:
You need to check that there is a rocblas/rocblas.h header before including it and then fallback to using rocblas.h when the header is missing.